contracts/bprotocol/Registry.sol

```diff
@@ -97,6 +97,15 @@ contract Registry is Ownable {
         emit RevokeDelegate(msg.sender, _avatar, delegatee);
     }
 
+    function delegateAndExecuteOnce(address delegatee, address payable target, bytes calldata data) payable external {
```

Comment:
visibility modifier should come before `payable`

Reply:
fixd